js/login.js

```diff
@@ -0,0 +1,74 @@
+// defould open login page
+window.location.href = "login.html";
```

Comment:
looks like constant page reload is caused by this line window.location.href = "login.html"; try to comment it out